tools/clang/include/clang/SPIRV/SpirvInstruction.h

```diff
@@ -524,8 +526,10 @@ class SpirvFunctionParameter : public SpirvInstruction {
 
 private:
   // When we turn on the rich debug info generation option, we want
-  // to keep the DebugLocalVariable instruction corresponding to
-  // this SpirvFunctionParameter instruction.
+  // to keep the function parameter information (like a local
+  // variable). Since DebugDeclare instruction maps a
+  // DebugLocalVariable instruction to OpVariable instruction, we
+  // keep a pointer to SpirvDebugDeclare in SpirvVariable.
```

Comment:
The comment seems to have been copied from the SpirvVariable class and not updated for this class.